.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 ---
 repos:
   - repo: https://github.com/UCL-MIRSG/.github
-    rev: v0.43.0
+    rev: v0.48.0
```

Comment:
this needs to be updated to 0.49.0 after merging this PR. Is there a way to always keep this up to date without need a follow-up PR?

Reply:
Not easily, it's not urgent, though. Just wait until next month and @renovatebot will run.